src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -457,7 +461,13 @@ If you have code that sets up global state, this state will need to take into ac
 </script>
 ```
 
-Module scripts, including those that add event listeners for `DOMContentLoaded`, are only ever executed once because the browser keeps track of which modules are already loaded. You must add listeners for a [Lifecycle event](#lifecycle-events) to re-execute scripts during client-side page navigation.
+Inline scripts can be reexecuted after every transition by adding the `data-astro-rerun` property. This will make the script `is:inline` implicitly, so it will not work for bundled scripts.
```

Comment:
```suggestion
Inline scripts can be re-executed after every transition by adding the `data-astro-rerun` property. This will make the script `is:inline` implicitly, so it will not work for bundled scripts.
```